src/acom_music_box/goMusicBox.bat

```diff
@@ -0,0 +1 @@
+python music_box_main.py configFile="C:\2024\MusicBox\music-box\tests\configs\analytical_config\my_config.json" outputDir="C:\2024\MusicBox\music-box\tests\configs\analytical_config"
```

Comment:
I agree with @K20shores - I would prefer this to be included in a simple example in the README as opposed to a script